src/utils/src/api-utils.ts

```diff
@@ -79,6 +79,14 @@ export const buildNotFoundResponse = (type: string, item: string) => {
   return buildFailureResponse(404, `Could not find the requested ${type} [${item}].`);
 };
 
+/**
+ * Builds an error failure API response object for when the requesting
+ * method is not allowed.
+ */
+export const buildMethodNotAllowedResponse = (msg: string) => {
+  return buildFailureResponse(405, msg);
```

Comment:
This method is a good idea and correct implementation, but I don't think we want to use 405 because of the way that it's supposed to be used: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405

Reply:
I have modified the changes and please let me know what you think!